src/main/java/uk/ac/ebi/eva/pipeline/io/readers/NonAnnotatedVariantsMongoReader.java

```diff
@@ -36,7 +36,7 @@ public NonAnnotatedVariantsMongoReader(String databaseName, String collectionsVa
                                            MongoConnection connection) throws UnknownHostException {
         super();
 
-        setMongo(MongoDBHelper.getMongoClient(connection));
+        setMongo(new MongoDBHelper().getMongoClient(connection));
```

Comment:
NonAnnotatedVariantsMongoReader should be harmonised with the rest of the readers/writers. The parent reader should be implemented again with the inclusion of template functionality.